libcodechecker/server/api/report_server.py

```diff
@@ -1939,66 +1953,153 @@ def massStoreRun(self, name, tag, version, b64zip, force):
                                                    max_run_count,
                                                    remove_run_count))
 
-        with util.TemporaryDirectory() as zip_dir:
-            # Unzip sent data.
-            unzip(b64zip, zip_dir)
+            # If the run can be stored, we need to lock it first.
```

Comment:
It would be better to move the run lock checking code to a separate function. This way the massStoreRun function will be much more readable.
https://github.com/whisperity/CodeChecker/blob/511a1f9daf29abcb480d0bde4fc2d09c2f34e338/libcodechecker/server/api/report_server.py#L1956-L2008

Reply:
Done.